### DISCUSSION TITLE
How to make AI analyze result of Function call

### DISCUSSION THREAD

**Discussion by jxpose** [spring-projects/spring-ai#1471] [Q&A]
Im trying to make my AI retrieve data and analyze the result, sales something.
I created a custom Function to retrieve the necessary data from the data source. 
Using the retrieved data, how can I make the AI to analyze the it?

**Comment by ogbozoyan** [comment]
Maybe add another AI layer which will be Analyze assistant

**Reply by jxpose** [reply]
Thank you for the insights. I'll try this
